### PR TITLE
Kevin/multiroute change

### PR DESCRIPTION
**Pull request by kevinchan159**
- `/multiroute` should `return the best available route for each destination specified from a single start location` but at the moment, it returns an array where each element is an array of routes for each destination. This PR fixes that.
- also added a command in `package.json` to start the backend locally so that we don't have to copy and paste the 3 commands every time

### CODE REVIEW

File: src/routers/MultiRouteRouter.js [hunk 1]
```diff
@@ -35,16 +36,21 @@ class MultiRouteRouter extends ApplicationRouter<Array<Object>> {
     }
 
     // multiple destinations given
```
Comment (yuna878) [inline src/routers/MultiRouteRouter.js]: Comment should address that the variable is an array of routes to each destinations

File: src/routers/MultiRouteRouter.js [hunk 2]
```diff
@@ -35,16 +36,21 @@ class MultiRouteRouter extends ApplicationRouter<Array<Object>> {
...
       departureTimeQuery,
       false,
     ));
 
-    return Promise.all(routes).then(val => val).catch((err) => {
-      throw LogUtils.logErr(err, routes, 'Could not get all specified routes');
+    // return the best route for each different destination
+    return Promise.all(multiRoutes).then((val) => {
+      const bestRoutes = val.map(routes => ((routes.length > 0) ? routes[0] : null));
+      AnalyticsUtils.assignRouteIdsAndCache(bestRoutes.filter(r => r !== null));
```
Comment (meganle) [inline src/routers/MultiRouteRouter.js]: You can do `bestRoutes.filter(Boolean)` to filter out faulty values

File: package.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "start:dev": "./node_modules/.bin/webpack --watch --env.NODE_ENV=development --progress --colors --mode=development --debug --devtool cheap-eval-source-map --output-pathinfo",
     "start:microservices": "docker-compose up -d ghopper ghopper-walking live-tracking map",
     "start:node": "docker-compose up node",
+    "start:backend": "npm run build:dev; npm run start:microservices; npm run start:node",
```
Comment (young-k) [inline package.json]: you only have to run the first two commands one time, and it should hot reload & the microservices persist in the background, so i don't think we need this

File: src/routers/MultiRouteRouter.js (hunk 2 above)
Comment (young-k) [reply src/routers/MultiRouteRouter.js]: aren't non null checks `!=` and not `!==`?

**Comment by young-k** [comment]
In the future, make sure to rename your PR titles as well

**Pull request merged**